playwright/src/main/java/com/microsoft/playwright/junit/Options.java

```diff
@@ -0,0 +1,307 @@
+package com.microsoft.playwright.junit;
+
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+public class Options {
+  private String baseUrl;
+  private Path storageState;
+  private ColorScheme colorScheme;
+  private GeoLocation geoLocation;
+  private String locale;
+  private List<String> permissions;
+  private String timezoneId;
+  private Viewport viewport;
+  private boolean acceptDownloads;
+  private Map<String, String> extraHTTPHeaders;
+  private HTTPCredentials httpCredentials;
+  private boolean ignoreHTTPSErrors;
+  private boolean offline;
+  private double actionTimeout;
+  private String browserName;
+  private boolean bypassCSP;
+  private Channel channel;
+  private boolean headless;
+  private String testIdAttribute;
+
+  public String getBaseUrl() {
+    return baseUrl;
+  }
+
+  public Options setBaseUrl(String baseUrl) {
+    this.baseUrl = baseUrl;
+    return this;
+  }
+
+  public Path getStorageState() {
+    return storageState;
+  }
+
+  public Options setStorageState(Path storageState) {
```

Comment:
won't we also need to expose `get` so that we can retrieve it when setting the options in playwright (assuming the extensions are in the impl package and Options is outside)?